robottelo/hosts.py

```diff
@@ -1597,7 +1597,8 @@ def install(self, installer_obj=None, cmd_args=None, cmd_kwargs=None):
         """General purpose installer"""
         if not installer_obj:
             command_opts = {'scenario': self.__class__.__name__.lower()}
-            command_opts.update(cmd_kwargs)
```

Comment:
this is failing when passing just `cmd_args`

Reply:
Lets better have another PR (cherrypickable to all) for this issue - https://github.com/SatelliteQE/robottelo/pull/14693